core/cli/src/plugin/options.ts

```diff
@@ -15,10 +17,28 @@ export const validatePluginOptions = (logger: Logger, config: ValidPluginsConfig
     }
 
     const pluginSchema = PluginSchemas[pluginId]
+
     if (!pluginSchema) {
       logger.silly(`skipping validation of ${pluginId} plugin as no schema can be found`)
+
+      // TODO:KB:20240412 remove legacyPluginOptions in a future major version
+      if (pluginOptions && pluginId in legacyPluginOptions) {
+        const movedToTask = legacyPluginOptions[pluginId]
+        invalidOptions.push([
```

Comment:
up to you if you want to change the implementation then, I don't feel as strongly if it will mostly improve errors rather than allowing backwards compatibility!